### DISCUSSION TITLE
External Storage - NAS

### DISCUSSION THREAD

**Discussion by micogg** [blakeblackshear/frigate#3646] [Ask A Question]
Hi all,

I'm looking to build a NAS and want to use Nextcloud for the ease of use for external access to pictures and other media.

My question is, would there be a way to store the Frigate snapshots and recordings to the Nextcloud instance?

Currently I'm running Frigate in a TrueNAS docker but I'd like to dedicate that machine to just Frigate.

**Comment by NickM-27** [comment]
Yea, you should be able to mount the next cloud as a standard docker volume
